include/openssl/ct.h

```diff
@@ -370,7 +373,7 @@ sct_validation_status_t SCT_get_validation_status(const SCT *sct);
  * Returns 0 if the SCT is invalid or could not be verified.
  * Returns -1 if an error occurs.
  */
-int SCT_validate(SCT *sct, const CT_POLICY_EVAL_CTX *ctx);
+__owur int SCT_validate(SCT *sct, const CT_POLICY_EVAL_CTX *ctx);
```

Comment:
Umm... and what happens if I validate an SCT and then modify some of its fields? Don't I end up in an inconsistent state?